Scripts/Services/Assistant/V1/AssistantService.cs

```diff
@@ -177,7 +177,11 @@ public bool Message(Callback<Dictionary<string, JObject>> callback, string works
                 }
             }
 
-            req.Headers["X-IBMCloud-SDK-Analytics"] = "service_name=conversation;service_version=V1;operation_id=Message";
+            foreach(KeyValuePair<string, string> kvp in Common.GetDefaultheaders("conversation", "V1", "Message"))
```

Comment:
Don't we need to do the same change for other services like [LT](https://github.com/watson-developer-cloud/unity-sdk/blob/master/Scripts/Services/LanguageTranslator/v3/LanguageTranslator.cs#L262) for instance.

Reply:
Yes! This is merging to the V3 branch though. I only have Assistant here so far. The rest are taken care of once I fully regenerate.